Client/Application/Migration.swift

```diff
@@ -14,6 +14,7 @@ extension Preferences {
     /// Migration preferences
     final class Migration {
         static let completed = Option<Bool>(key: "migration.completed", default: false)
+        static let syncOrderCompleted = Option<Bool>(key: "syncorder.migration.completed", default: false)
```

Comment:
Small nit (😅): Considering all the keys it'd probably be better to name this `migration.sync-order.completed`

It doesn't _really_ matter though heh.